app/src/components/integrations/elements/integrationSettings/integrationForm/integrationForm.scss

```diff
@@ -38,6 +38,19 @@
   justify-content: center;
   width: 100%;
   margin-top: 5px;
+
+  svg path {
+    fill: $COLOR--e-300;
+  }
+
+  /* stylelint-disable */
+  [class*="with-popover"] {
```

Comment:
I am not quite sure, but looks like we can solve this using another approach

app/src/components/integrations/elements/bts/btsPropertiesForIssueForm/btsPropertiesForIssueForm.jsx

Find IconShow component and wrap Parser(icon) with span f.e.
And you can change necessary properties without important and such complex selectors

Reply:
Done.